pyproject.toml

```diff
@@ -62,3 +62,90 @@ strict_equality = true
 [[tool.mypy.overrides]]
 module = ["django.*"]
 ignore_missing_imports = true
+
+[tool.ruff]
+line-length = 120
+target-version = "py38"
+
+[tool.ruff.lint]
```

Comment:
question: why this set of lint rules?

Reply:
These are the ones that currently pass.  Future PRs can uncomment rules that might be of interest.  We do not want this PR to contain so many changes that it slows the review process.